src/components/v2v3/V2V3Project/V2V3PayButton/V2V3ConfirmPayModal/V2V3ConfirmPayModal.tsx

```diff
@@ -95,11 +95,10 @@ export function V2V3ConfirmPayModal({
     plural: true,
   })
 
-  let nftRewardTier: NftRewardTier | undefined
+  let nftRewardTiers: NftRewardTier[] | undefined
```

Comment:
**suggestion** this can be combined into one line

```suggestion
  const nftRewardTiers: rewardTiers?.filter // etc.
```

Reply:
Ah yep lol whack